lib/libc/arcmwdt/include/fcntl.h

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (c) 2021 Synopsys
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef ZEPHYR_LIB_LIBC_ARCMWDT_INCLUDE_FCNTL_H_
+#define ZEPHYR_LIB_LIBC_ARCMWDT_INCLUDE_FCNTL_H_
+
+#define O_CREAT    0x0200
```

Comment:
Aren't these define in MW sys/fctl?
Can you simply define them here, without ensuring that the MW implementation isn't using other constants?

Reply:
Fixed. Now we only define O_NONBLOCK which is missing in MWDT fcntl.h header. We also check that our O_NONBLOCK value doesn't conflict with any other file IO defines which came from MWDT fcntl.h header.

Reply:
Nice solution, thanks.